modules/darwin/community-builder/default.nix

```diff
@@ -1,6 +1,16 @@
+{ pkgs, ... }:
 {
   imports = [
     ./packages.nix
     ./users.nix
   ];
+
+  programs.zsh = {
```

Comment:
Moved this from darwin/common to darwin/community-builder. 

Zsh is only enabled on all the darwin machines because is the the macos default.